### DISCUSSION TITLE
Is it possible to add a QR code generator?

### DISCUSSION THREAD

**Discussion by avishka-codimite** [react-native-tvos/react-native-tvos#339] [Q&A] [answered]
I have to implement a QR code generator for my project. so I decided to go with these two packages [react-native-svg](https://www.npmjs.com/package/react-native-svg) and [react-native-qrcode-svg](https://www.npmjs.com/package/react-native-qrcode-svg). but seems to be `react-native-qrcode-svg` package not support to this `react-native-tvos.`  So if there any package to do this task, please let me know.

**Comment by nriccar** [comment, marked as answer]
You could solve this generating the QR from another environment, backend, and send it as an image trough an API to the app. I solved it that way!

**Reply by avishka-codimite** [reply]
I found this [react-qr-code](https://www.npmjs.com/package/react-qr-code) package and seems to be it works without any issue.

**Reply by nriccar** [reply]
Nice!
